src/sensors/icm20948sensor.cpp

```diff
@@ -359,174 +254,263 @@ void ICM20948Sensor::motionSetup() {
        m_Logger.fatal("Failed to reset FIFO");
         return;
     }
+}
 
-#if LOAD_BIAS
-    // Initialize the configuration
+void ICM20948Sensor::connectSensor()
+{
+    #ifdef DEBUG_SENSOR
+        imu.enableDebugging(Serial);
+    #endif
+    // SparkFun_ICM-20948_ArduinoLibrary only supports 0x68 or 0x69 via boolean, if something else throw a error
+    boolean tracker = false;
+    
+    if (addr == 0x68) {
+        tracker = false;
+    } else if (addr == 0x69)
     {
```

Comment:
Please put that curly a line above, if you're doing in the rest of function as well.